bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/openthermgateway.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="openthermgateway"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="openthermgateway">
+		<label>OpenTherm Gateway</label>
+		<description>OpenTherm Gateway</description>
+		<channels>
+			<channel id="sendcommand" typeId="sendcommand"/>
+		</channels>
+		<properties>
+			<property name="version">2.1.4</property>
```

Comment:
Could you update the version number of the binding ? I use this when providing support as bindings don't otherwise seem to have their own version number. I moved to major 2 with the introduction of bridge and separate things, so the next version number would be 2.2.0

Reply:
Otherwise LGTM, thanks for the effort you've put into completing this PR

Reply:
Ok.